### ISSUE TITLE
Voice changes too much

### ISSUE THREAD

**Issue by geb6426** [suno-ai/bark#577]
They aren't same person's voice everytime. How to change it to one person's voice instead of a different voice everytime python -m bark --text "Hello, my name is Suno." --output_filename "example.wav"

**Comment by JonathanFly** [comment]
You need to specify a voice, or a history prompt. 
```
python -m bark --text "Hello, my name is Suno and I am voice 6." --history_prompt 'v2/en_speaker_6' --output_filename "06_example.wav"
```

**Comment by PhilipAmadasun** [comment]
@JonathanFly How do I do this in python code. Doe the "geenrate_audio" function have this as an argument?

**Comment by vieirinhasantana** [comment]
@PhilipAmadasun There is a parameter you can pass to set the voice.

audio_array = generate_audio("TEXT_PROMPT", history_prompt="v2/en_speaker_6" )

The voices available here.
https://suno-ai.notion.site/8b8e8749ed514b0cbf3f699013548683?v=bc67cff786b04b50b3ceb756fd05f68c

**Comment by JonathanFly** [comment]
> @PhilipAmadasun There is a parameter you can pass to set the voice.
> 
> audio_array = generate_audio("TEXT_PROMPT", history_prompt="v2/en_speaker_6" )
> 
> The voices available here. https://suno-ai.notion.site/8b8e8749ed514b0cbf3f699013548683?v=bc67cff786b04b50b3ceb756fd05f68c

This is correct, but there are many other voices you can use. You can save any random Bark output as a new voice .npz file and use it in the future.

**Comment by PhilipAmadasun** [comment]
@JonathanFly Are there parameters to make the model run faster for  real time?
